ltr/concourse/deploy.py

```diff
@@ -15,7 +15,7 @@
 
 endpoint_version = f"-{model_tag}" if model_tag else ""
 s3_bucket = os.getenv("S3_BUCKET", f"govuk-{govuk_environment}-search-relevancy")
-default_initial_instance_count = int(os.getenv("INITIAL_INSTANCE_COUNT", 3))
+default_initial_instance_count = int(os.getenv("INITIAL_INSTANCE_COUNT", 4))
```

Comment:
This won't actually scale up the number of instances we currently have - `default_initial_instance_count` is only used if an endpoint doesn't currently exist (see line 38)

Reply:
Thanks @barrucadu, I've made the change through the AWS UI, too.